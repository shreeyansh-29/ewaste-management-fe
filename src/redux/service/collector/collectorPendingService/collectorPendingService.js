import {COLLECTOR_REQUEST_PENDING} from "../../../../container/constant/constant";
import api from "../../../../core/utilities/httpProvider";

export const collectorPendingService = () => {
  return api.get(COLLECTOR_REQUEST_PENDING);
};
