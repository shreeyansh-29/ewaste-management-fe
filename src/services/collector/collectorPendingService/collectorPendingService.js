import {COLLECTOR_REQUEST_PENDING} from "../../../container/constant/constants";
import api from "../../../core/utilities/httpProvider";

export const collectorPendingService = () => {
  return api.get(COLLECTOR_REQUEST_PENDING);
};
