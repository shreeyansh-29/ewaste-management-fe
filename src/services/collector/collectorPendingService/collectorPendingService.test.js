/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import {collectorPendingService} from "./collectorPendingService";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing collectorPendingService", () => {
  const Service = collectorPendingService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
