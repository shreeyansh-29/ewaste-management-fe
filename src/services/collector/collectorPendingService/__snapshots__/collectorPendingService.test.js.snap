// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing collectorPendingService 1`] = `<Component />`;
